src/Test/Perf/Runner/Program.cs

```diff
@@ -52,6 +52,24 @@ public static void Main(string[] args)
             }
         }
 
+        private static void Cleanup()
```

Comment:
Is this called anywhere? I dont see a callsite for this method. May be I missed it